cinema_service/urls.py

```diff
@@ -1,6 +1,8 @@
+from debug_toolbar.toolbar import debug_toolbar_urls
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-]
+    path("api/", include("cinema.urls", namespace="cinema")),
+] + debug_toolbar_urls()
```

Comment:
The `debug_toolbar_urls()` is added directly to `urlpatterns`. It's recommended to conditionally include debug toolbar URLs only when `DEBUG` is `True` to avoid exposing debug information in production.